packages/components/link/src/link.js

```diff
@@ -14,16 +14,27 @@ const getLinkStyles = (props, theme) => {
     ...theme,
   };
 
+  const getColorValue = (tone) => {
+    if (tone === 'inverted')
+      return overwrittenVars.fontColorForTextWhenInverted;
+    return overwrittenVars.colorPrimary;
+  };
+  const getActiveColorValue = (tone) => {
+    if (tone === 'inverted')
+      return overwrittenVars.fontColorForTextWhenInverted;
```

Comment:
Is it correct to use the same color for the hover/active state?

Reply:
This is agreed with UX as well.